xcodeproj/internal/bazel_integration_files/copy_outputs.sh

```diff
@@ -19,6 +19,21 @@ else
       suffix=/Payload
     fi
 
+    if [[ -n ${BAZEL_OUTPUTS_DSYM:-} ]]; then
+      read -a dsym_files <<< "$BAZEL_OUTPUTS_DSYM"
```

Comment:
I don't think this will be good enough. I really think you need to go the `xargs` route like I did for `PREVIEW_FRAMEWORK_PATHS`. This way won't handle spaces correctly. The entries need to be quoted items.